cmd/tink-worker/cmd/root.go

```diff
@@ -92,7 +89,6 @@ func NewRootCommand(version string, logger log.Logger) *cobra.Command {
 	rootCmd.Flags().Int("max-retry", defaultRetryCount, "Maximum number of retries to attempt (MAX_RETRY)")
 	rootCmd.Flags().Int64("max-file-size", defaultMaxFileSize, "Maximum file size in bytes (MAX_FILE_SIZE)")
 	rootCmd.Flags().Bool("capture-action-logs", true, "Capture action container output as part of worker logs")
-	rootCmd.Flags().Bool("tinkerbell-tls", true, "Connect to server via TLS or not (TINKERBELL_TLS)")
```

Comment:
I believe we still want to have this optionally enabled for tink-worker. We could conceivably run the tink server behind a TLS-terminated ingress controller and then we'd want to have the tink-worker <-> server communication with TLS.

Reply:
That's fair. I figured we'd reimplement across the board in a saner way but no harm maintaining it until then.

Reply:
Added TLS back to server, worker and virtual worker as before.

Reply:
Ah sorry, I actually wasn't suggesting we put it all back. Only for tink worker to use TLS. And not TLS that requires a cert to be passed in. So only sever side TLS from a know root CA's.

Reply:
So we want to officially announce no TLS support in Tink Server, for now? That's the direction Hegel has gone also and I expect the rest of the stack to go because the TLS implementations felt a little hacky.

Long term I think it would be good to support TLS configuration as an opt-in, thoughts?

Reply:
In tink server, long term, I don't know if we should support TLS termination. Definitely a conversation to have though. I think, for sure, some detailed documentation needs to be created to describe the different TLS termination options and highlight that TLS communication between server and worker can still be achieved without functionality to terminate in tink server.

Reply:
Sounds good. Will strip it out of this PR.

Reply:
Stripped TLS from the server